.github/workflows/pr.yml

```diff
@@ -48,14 +48,14 @@ jobs:
           set +ex
           echo "## Improvements are listed below" | tee -a ${GITHUB_STEP_SUMMARY}
           echo "\`\`\`diff" >> ${GITHUB_STEP_SUMMARY}
-          diff -u0 branch.errors main.errors | grep '^+[^+]' | tee -a ${GITHUB_STEP_SUMMARY}
+          diff -u0 branch.errors main.errors | grep '^+[^+]' | sed -e 's/ERROR/FIXED/' | tee -a ${GITHUB_STEP_SUMMARY}
           echo "\`\`\`" >> ${GITHUB_STEP_SUMMARY}
           echo "## Regressions are listed below" | tee -a ${GITHUB_STEP_SUMMARY}
           echo "\`\`\`diff" >> ${GITHUB_STEP_SUMMARY}
           diff -u0 branch.errors main.errors | grep '^-[^-]' | tee -a ${GITHUB_STEP_SUMMARY}
           echo "\`\`\`" >> ${GITHUB_STEP_SUMMARY}
-          if ! diff -q main.errors branch.errors;  then
-             echo "> Fix the regressions listed above (starting with a '-')" | tee -a ${GITHUB_STEP_SUMMARY}
+          if [[ $(diff -u0 branch.errors main.errors | grep -c '^-[^-]') != 0 ]];  then
```

Comment:
why not just checking the status of the grep command?

Reply:
Because the comparison has to be on the diff.

Is possible the branch.errors still contain errors from main.errors, those are the improvements.

Reply:
even with a pipe, it works as it checks the status of the last command.

Reply:
Got it, I did the grep -q, same effect as the pipe